.github/workflows/ci.yml

```diff
@@ -0,0 +1,24 @@
+name: Avocado Gating CI
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    name: Logical, stylistic, analytical and formatter checks
+    runs-on: ubuntu-latest
```

Comment:
Everything that can be pinned down on a CI environment should.  Can you double check that this won't be automatically updated when a new Ubuntu release is out?  Also, what are the options for distros here?

Reply:
Yes, I agree that we should be pinning this. I will change that.

About the distros available right now:


Windows Server 2019 | windows-latest or windows-2019
-- | --
Ubuntu 20.04 | ubuntu-latest or ubuntu-20.04
Ubuntu 18.04 | ubuntu-18.04
Ubuntu 16.04 | ubuntu-16.04
macOS Big Sur 11.0 | macos-11.0
macOS Catalina 10.15 | macos-latest or macos-10.15

They are vms, but looks like we can also run on dockers. Custom runners are also possible.